benchmark/common.js

```diff
@@ -203,15 +210,29 @@ Benchmark.prototype.end = function(operations) {
 
 Benchmark.prototype.report = function(value) {
   var heading = this.getHeading();
-  if (!silent)
-    console.log('%s: %s', heading, value.toFixed(5));
+
+  if (!silent) {
+    if (outputFormat == 'default')
+      console.log('%s: %s', heading, value.toFixed(5));
+    else if (outputFormat == 'csv')
+      console.log('%s,%s', heading, value.toFixed(5));
+  }
 
   process.exit(0);
 };
 
 Benchmark.prototype.getHeading = function() {
   var conf = this.config;
-  return this._name + ' ' + Object.keys(conf).map(function(key) {
-    return key + '=' + conf[key];
-  }).join(' ');
+
+  if (outputFormat == 'default') {
+    return this._name + ' ' + Object.keys(conf).map(function(key) {
+      return key + '=' + conf[key];
+    }).join(' ');
+  } else if (outputFormat == 'csv') {
+    return this._name + ',' + Object.keys(conf).map(function(key) {
+      return conf[key];
+    }).join(',');
+  } else {
+    throw new Error('OUTPUT_FORMAT set to invalid value');
```

Comment:
Suggestion: check outputFormat at the top, right after retrieving it from the env.